menu_platinum.rb

```diff
@@ -0,0 +1,70 @@
+# Lila Garcia
+# Random Menu Generator -- Dessert (menu items are geneated by user input)
+
+# Create empty arrays of menu items
+
+# adjectives
+descriptors = []
+
+# cooking prep
+style = []
+
+# foods
+foods = []
+
+# Prompts for user input
+puts "Hello! I am here to build you a menu."
+puts "Please enter the number of menu items you would you like to see." 
+array_count = $stdin.gets.chomp.to_i 
```

Comment:
I'd recommend adding some sort of error handling to ensure that the user has actually entered a numeric value